Source/WebCore/css/CSSViewTransitionRule.cpp

```diff
@@ -54,6 +54,19 @@ StyleRuleViewTransition::StyleRuleViewTransition(Ref<StyleProperties>&& properti
     : StyleRuleBase(StyleRuleType::ViewTransition)
 {
     m_navigation = toViewTransitionNavigationEnum(properties->getPropertyCSSValue(CSSPropertyNavigation));
+
+    if (auto value = properties->getPropertyCSSValue(CSSPropertyTypes)) {
+
+        auto processSingleValue = [&](const CSSValue& currentValue) {
+            if (currentValue.isCustomIdent())
```

Comment:
Can it be anything else than a custom ident at this stage?

Reply:
It can be the not-custom 'none' ident.